controllers/apply/form-router-next/joi-extensions/bank-numbers.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+const { isString } = require('lodash');
+
+module.exports = function bankNumbers(joi) {
```

Comment:
Not 100% sure this needs to be written as a custom validator. Curious if `joi.string().replace(/\D/g, '').length(5)` has the same effect? At the least both of the custom rules are repeating what length does, so it feels like there should be a way to avoid that duplication.